components/02-elements/table/_table.scss

```diff
@@ -13,7 +13,7 @@ $table-radius: $border-radius;
     font-size: $font-size-small;
 
     &--odd-even {
```

Comment:
move modifiers UNDER the default code - b/c their role is to overwrite default module code. Please do it everywhere in separate commit.

Reply:
👍